### PR TITLE
Fixes typos

### PR DESCRIPTION
**Pull request by vchabri**

### CODE REVIEW

File: python/doc/examples/meta_modeling/general_purpose_metamodels/plot_taylor_approximation.py [hunk 1]
```diff
@@ -35,13 +35,13 @@
...
 # %%
-# create a linear (first order) Taylor approximation
+# Create a linear (first order) Taylor approximation.
 algo = ot.LinearTaylor(x0, model)
 algo.run()
 responseSurface = algo.getMetaModel()
 
 # %%
-# plot 2nd output of our model with x1=x0_1
+# Plot 2nd output of our model with x1=x0_1.
```
Comment (josephmure) [inline python/doc/examples/meta_modeling/general_purpose_metamodels/plot_taylor_approximation.py:44]:
```suggestion
# Plot the second output of our model with :math:`x_1=x_{0,1}`.
```
To keep notations consistent with the first equation.

File: python/doc/examples/meta_modeling/general_purpose_metamodels/plot_taylor_approximation.py [hunk 2]
```diff
@@ -56,20 +56,20 @@
...
 # %%
-# create a quadratic (2nd order) Taylor approximation
+# Create a quadratic (second order) Taylor approximation.
 algo = ot.QuadraticTaylor(x0, model)
 algo.run()
 responseSurface = algo.getMetaModel()
 
 # %%
-# plot 2nd output of our model with x1=x0_1
+# Plot second output of our model with x1=x0_1.
```
Comment (josephmure) [inline python/doc/examples/meta_modeling/general_purpose_metamodels/plot_taylor_approximation.py:72]:
```suggestion
# Plot the second output of our model with :math:`x_1=x_{0,1}`.
```
Same.

**Review by josephmure — CHANGES_REQUESTED** [review]

**Comment by josephmure** [comment]
This goes beyond what @mbaudin47 requested in #1935. Good job!

**Comment by josephmure** [comment]
I guess another PR will be required to fix those additional typos.

**Comment by jschueller** [comment]
Yes, sorry

**Comment by mbaudin47** [comment]
It seems that the backslashes were not taken into account:

image

**Comment by josephmure** [comment]
@mbaudin47 CircleCI failed to publish the doc when #1977 was merged, otherwise we would see that all changes were taken into account:

https://8612-40717169-gh.circle-artifacts.com/0/html/auto_meta_modeling/general_purpose_metamodels/plot_taylor_approximation.html#sphx-glr-auto-meta-modeling-general-purpose-metamodels-plot-taylor-approximation-py

**Comment by mbaudin47** [comment]
Ok.

**Pull request merged**